test/performance/array/distCreate-large.ml-perf.graph

```diff
@@ -0,0 +1,5 @@
+perfkeys: blockDom:, blockArr:, cyclicArr:, cyclicDom:, blockCyclicDom:, blockCyclicArr:
+files: distCreate-large 
+ylabel: Time (seconds)
+graphname: distCreate-small
```

Comment:
I don't think we use/need graphname anymore  (I'd look at some of the other `ml-perf.graph` files as a template)

Reply:
Oh, this also needs to be `distCreate-small.dat` (For some reason we require the .dat extension here, but not in compopts/execopts ) -- FWIW I usually don't remember this off the top of my head. I normally copy an existing test as a template, and I also look at the graph to make sure it looks right